scripts/sign-multiarch.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euo pipefail
+
+REPO=${IMG_REPO:-docker.io/mongodb/mongodb-atlas-kubernetes-operator-prerelease}
```

Comment:
Where is `IMG_REPO` coming from?

Reply:
Is a env var input. Having `set -euo` above will make the script fail if we omit the env var.